modules/ROOT/pages/provisioning-libvirt.adoc

```diff
@@ -10,6 +10,8 @@ You also need to have access to a host machine with `libvirt`. The examples belo
 
 TIP: If running on a host with SELinux enabled (use the `sestatus` command to check SELinux status), make sure your OS image and Ignition file are labeled as `svirt_home_t`. You can do this by placing them under `~/.local/share/libvirt/images/` or running `chcon -t svirt_home_t /path/to/file`.
 
+TIP: If running on a host with AppArmor enabled, you may need to change the AppArmor profile for libvirt. Add `/absolute/path/to/ignition/file.ign rk,` to the `LIBVIRT_TEMPLATE` profile that can be found in `/etc/apparmor.d/libvirt/TEMPLATE.qemu`.
```

Comment:
I am a bit unsure about that because libvirt relies on dynamic profiles created from the `/etc/apparmor.d/libvirt/TEMPLATE.qemu` template and removed upon VM removal. Local overrides are supposed to override specific profiles from what I understood.

In either case, I'll try to test it and revert with the results.

Reply:
libvirt itself does not uses those profiles directly, it's AppArmor configuration logic. Local overrides are there to add to distribution provided profiles, which in the case of AppArmor are still stored in /etc unfortunately.

Reply:
@nrvnrvn Any updates on this?